### ISSUE TITLE
Provide some documentation or features and license

### ISSUE THREAD

**Issue by senpl** [adkrueger/Not_Just_Dance#1]
I just do not know what's this project is about. Much more info would be nice. I would also like to note what license it got.

**Comment by adkrueger** [comment]
Hi @senpl, this project was created for a hackathon last year. We used the posenet2 library to see a user's position with the camera, and attempted to see how well this matched the "character" they were trying to mimic. If you've ever played or seen the game "Just Dance", it's meant to be like that. However, since we ran out of time during the hackathon, we couldn't quite get the scoring to work and ended up making it so the game scored the player based on how well they match the pose the program is displaying.

.json files were used to keep track of what positions the "dance" was supposed to display, and our other .js files worked with scoring, drawing the user/computer players on the screen, and getting posenet2 working for us. Again, this was just for fun for a hackathon so our commits, documentation, and filenames are rather informal.

I am unsure about licensing; is there an easy way to find that out?
